source-zendesk-support/tests/snapshots/snapshots__discover__capture.stdout.json

```diff
@@ -3885,7 +3885,7 @@
       "stream": "tickets",
       "syncMode": "incremental",
       "cursorField": [
-        "updated_at"
```

Comment:
I assume this doesn't actually matter, since the `"after_cursor"` value is hard coded in the connector

Reply:
Yep. Before, the `cursorField` was being used to retrieve a timestamp from one of the streamed documents. But since these streams' cursors are now being stored outside of the documents, we don't need `"updated_at"` to know where to look in the documents any more. 

The new `cursorField` is just the key we use when accessing the persisted `state`, and I gave it the same name as the response body's property `after_cursor` that contains the cursor.